network/src/commonMain/kotlin/com/wire/kalium/network/api/contact/search/ContactSearchRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.kalium.network.api.contact.search
+
+import kotlinx.serialization.SerialName
+
+data class ContactSearchRequest(
+    @SerialName("q")
+    val searchQuery: String,
+    @SerialName("domain")
+    val domain: String? = null,
```

Comment:
```suggestion
    val domain: String
```
According to swagger
> Note: This is optional only for backwards compatibility, future versions will mandate this.